openprescribing/measures/definitions/ktt13_nsaids_ibuprofen.json

```diff
@@ -1,21 +1,16 @@
 {
-  "name": "Non-preferred NSAIDs and COX-2 inhibitors (KTT13)",
-  "title": [
-    "KTT13: NSAIDs (including COX-2 inhibitors) - percentage items excluding naproxen and ibuprofen"
-  ],
-  "description": [
-    "Number of prescription items for all NSAIDs excluding ibuprofen and naproxen",
-    "as a percentage of the total number of prescription items",
-    "for all NSAIDs."
-  ],
+  "name": "Non-preferred NSAIDs and COX-2 inhibitors",
+  "title": "NSAIDs (including COX-2 inhibitors) - percentage items excluding naproxen and ibuprofen",
+  "description": "Number of prescription items for all NSAIDs excluding ibuprofen and naproxen as a percentage of the total items for all NSAIDs",
   "numerator_short": "NSAID items (excl naproxen and ibuprofen)",
   "denominator_short": "NSAID items",
+  "y_label": "Percentage of all NSAID items which aren't naproxen and ibuprofen",
   "why_it_matters": [
     "There have been a number of concerns about the safety of a number of anti-inflammatory drugs, including COX-2 inhibitors and some ",
     "'traditional' NSAIDS, for their effects on the stomach, kidneys and the risk of blood clots.  ",
-    "<a href='https://www.nice.org.uk/advice/ktt13/resources/nonsteroidal-antiinflammatory-drugs-58757951055301'>NICE recommends that ",
-    "as ibuprofen and naproxen are two of the drugs with a lower level of risk,</a> doctors should prescribe these to patients ",
-    "before trying the others."
+    "<a href='https://cks.nice.org.uk/topics/nsaids-prescribing-issues/'>NICE CKD recommends that</a>",
```

Comment:
Change wording - do we need to exclude 600mg tablets (and in fact ONLY have tablets as measure)

Reply:
Interesting point, personally I'm not sure we need to change though. A person might just take 600mg tablets as 'ONE TWICE DAILY', for example if their pain is ok in the evening and they just need analgesia whilst more mobile during the day. Even patients on 200/400mg tablets could still be taking >1200mg TDD.
It might be difficult for us to discern the TDD with good reliability in this measure (definitely impossible for liquids), so I'd be tempted to say that dose considerations are out of scope for this measure.
We do still know that Ibuprofen/Naproxen are first line options, and I think still valid to look at their usage, independently of dose.